Tag component (#651)
microsoft/atlas-design: site/src/components/tag.md

			/>
		</svg>
	</span>
	<span>Tag</span>
</span>
```

### Sizes

The following classes are available for resizing: `tag-sm`, `tag-lg`.

```html
<span class="tag tag-sm">Tag</span>
<span class="tag">Tag</span>
<span class="tag tag-lg">Tag</span>
```

### Types

By default, tags are secondary outlined tags.

| State    | Class              | Example                                  |
| -------- | ------------------ | ---------------------------------------- |
| Outlined | `.tag`             | <span class="tag">Tag</span>             |
| Filled   | `.tag.tag-filled`  | <span class="tag tag-filled">Tag</span>  |
| Primary  | `.tag.tag-primary` | <span class="tag tag-primary">Tag</span> |

### Dismissable tags

> The only possible action that can be taken is dismissal.

Author: How do you feel about that:

Dismissible tags are tags that can be removed by the user. These tags include a close icon alongside the tag's content and the entire tag is treated as a click/tap target for the purposes of dismissal.

> The only possible action that can be taken is dismissal.

Reviewer: I like it!

> The only possible action that can be taken is dismissal.

Author: Updated

| Primary  | `.tag.tag-primary` | <button type="button" class="tag tag-primary"><span>Tag</span><span class="tag-close"><span class="icon" aria-hidden="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448" class="fill-current-color"><path d="M269.254 224l137.373 137.373-45.254 45.254L224 269.254 86.627 406.627l-45.255-45.254L178.746 224 41.373 86.627l45.255-45.255L224 178.746 361.373 41.373l45.254 45.255L269.254 224z"/></svg></span></span></button> | <button type="button" class="tag tag-primary is-hovered"><span>Tag</span><span class="tag-close"><span class="icon" aria-hidden="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448" class="fill-current-color"><path d="M269.254 224l137.373 137.373-45.254 45.254L224 269.254 86.627 406.627l-45.255-45.254L178.746 224 41.373 86.627l45.255-45.255L224 178.746 361.373 41.373l45.254 45.255L269.254 224z"/></svg></span></span></button> |

### Interactive tag

> In addition to dismissibility, interaction tags can have one other action that can provide more information or actions related to what the tag represents.

Reviewer: For my own understanding - will the popover-style interaction for interactive tags be only non-dismissal interaction style for tags? Could, for instance, there ever been an "interactive" tag that's just a `<button>` and not a popover summary?

Author: I don't think so, check out fluent's figma  - https://www.figma.com/design/GvIcCw0tWaJVDSWD4f1OIW/Fluent-2-web?node-id=133557-2279&t=q0Q551AFJqataUFz-0

Reviewer: In that case, I might suggest a rewording here:

```suggestion
Interactive tags can be used to open a popover that provides more information or contextual actions related to the tag's subject. Interactive tags can also be dismissible.
```

Author: Updated

> To do this, nest the [popover component](./popover.md) inside a tag.

Reviewer: ```suggestion
To make an interactive tag, nest the [popover component](./popover.md) inside a tag.
```

Author: Updated
